[2.9 ACRA] BootService.scheduleDeckReminder crash, null Collection on startup

https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/e7a7bb70-f978-449e-83ff-5a595571ae99

```
java.lang.RuntimeException: Unable to create application com.ichi2.anki.AnkiDroidApp: java.lang.NullPointerException: Attempt to invoke virtual method 'com.ichi2.libanki.Decks com.ichi2.libanki.Collection.getDecks()' on a null object reference
at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5934)
at android.app.ActivityThread.-wrap1(Unknown Source:0)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1680)
at android.os.Handler.dispatchMessage(Handler.java:105)
at android.os.Looper.loop(Looper.java:164)
at android.app.ActivityThread.main(ActivityThread.java:6710)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:770)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'com.ichi2.libanki.Decks com.ichi2.libanki.Collection.getDecks()' on a null object reference
at com.ichi2.anki.services.BootService.scheduleDeckReminder(BootService.java:2)
at com.ichi2.anki.services.BootService.onReceive(BootService.java:3)
at com.ichi2.anki.AnkiDroidApp.onCreate(AnkiDroidApp.java:24)
at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1119)
at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5931)
```

--- a/AnkiDroid/src/main/java/com/ichi2/anki/services/BootService.java
+++ b/AnkiDroid/src/main/java/com/ichi2/anki/services/BootService.java
@@ -33,6 +33,11 @@ public class BootService extends BroadcastReceiver {
         if (!CollectionHelper.hasStorageAccessPermission(context)) {
             return;
         }
+        // There are cases where the app is installed, and we have access, but nothing exist yet
+        if (CollectionHelper.getInstance().getCol(context) == null
+                || CollectionHelper.getInstance().getCol(context).getDecks() == null) {
+            return;
+        }
 
         scheduleDeckReminder(context);
         scheduleNotification(context);

